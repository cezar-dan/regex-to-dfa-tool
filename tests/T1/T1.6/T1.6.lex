 
SPACE
0
0,' ',1
1,' ',2
1

d
DS
0
0,'d',1
1,'d',1
1

ab
ABS
0
0,'a',1
0,'b',2
1,'a',2
1,'b',3
3,'a',1
3,'b',2
3 0

abc
ABC
0
0,'a',1
0,'b',2
0,'c',2
1,'a',2
1,'b',3
1,'c',2
3,'a',2
3,'b',2
3,'c',4
4,'a',2
4,'b',2
4,'c',2
4

abcd
APLUSBCD
0
0,'a',1
0,'b',2
0,'c',2
0,'d',2
1,'a',1
1,'b',3
1,'c',2
1,'d',2
3,'a',2
3,'b',2
3,'c',4
3,'d',2
4,'a',2
4,'b',2
4,'c',2
4,'d',5
5,'a',2
5,'b',2
5,'c',2
5,'d',2
5

bc
BORCS
0
0,'b',1
0,'c',2
1,'b',1
1,'c',2
2,'b',1
2,'c',2
2 1 0

bcd
BCSD
0
0,'b',1
0,'c',2
0,'d',3
1,'b',2
1,'c',4
1,'d',2
3,'b',2
3,'c',2
3,'d',2
4,'b',1
4,'c',2
4,'d',3
3

acd
DSTARACS
0
0,'a',1
0,'c',2
0,'d',3
1,'a',2
1,'c',4
1,'d',2
3,'a',1
3,'c',2
3,'d',3
4,'a',1
4,'c',2
4,'d',2
4