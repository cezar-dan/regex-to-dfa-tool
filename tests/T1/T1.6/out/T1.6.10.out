SPACE  
DS d
SPACE  
DSTARACS dacac
APLUSBCD aaabcd
BCSD bcd
BORCS bbb
DSTARACS dddac